dashboard_showcase.rb

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env ruby
+
+def load_rails # TODO: extract to script
+  ENV['RAILS_ENV'] = ARGV[0] || 'development'
+  require './config/environment'
+end
+
+# --------------------
+# functions and consts
+# --------------------
+
+def push_metric (prov, timestamp, interval = "daily")
+  prov.metric_rollups << MetricRollup.create(
```

Comment:
If you want you can save the ```prov.metric_rollups <<``` with adding resource_type and resource_id to the create.
I'm ok with both approaches